u3-views/app.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express')
+const expHbs = require('express-handlebars')
+const { courseDetails, courseUnits, courseStudents } = require('./features/courseController')
+const { index } = require('./features/indexController')
+const { Video, Videohome } = require('./features/videoController')
```

Comment:
โปรดใช้ `lowerCamelCase` สำหรับชื่อตัวแปรของคุณ!
Please use `lowerCamelCase` for your variable names!

```js
video
videoHome
```